### PR TITLE
feat: blog + docs for csv import/export support

### PR DESCRIPTION
**Pull request by Anty0**

### CODE REVIEW

**Comment by Anty0** [comment]
Note: [This change](https://github.com/tolgee/documentation/commit/2914794c05de7fae01a670fffcb4e345e5e10af9#diff-e9d5765cbdd0631fa448cf2b7996921177c99c676b6208c954d1864857ed446aR1) seem to be a reason why the build test fails

File: blog/2024-10-08-csv-format-support-released.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+We are excited to announce that Tolgee now supports the [CSV format](/platform/formats/csv), a simple format for table-like data. In Tolgee, you can import and export your localization data in CSV format.
+
+<!--truncate-->
+
+One less format to worry about! The support for the CSV format has landed in Tolgee. For those who store their localization data in CSV files, you can now import and export your data and manage them in Tolgee.
+
+When conversion to ICU placeholders is enabled, you can choose from various placeholder & message formats to use with CSV.
+
+Do you have any questions about those new features? Join the [Tolgee Slack community](https://Tolg.ee/slack) and let us know!
```
Comment (JanCizmar) [inline blog/2024-10-08-csv-format-support-released.mdx:33]: Add "read more in the docs".

File: platform/formats/csv.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### Exporting via REST API
+To export to CSV via REST API, you have to set the following parameters:
+```json
+{
+  "format": "CSV",
+}
+```
+
+Example with CURL:
```
Comment (JanCizmar) [inline platform/formats/csv.mdx:42]: Actually the CURL is a bit outdated, since we have the CLI.

**Review by JanCizmar — CHANGES_REQUESTED** [review]

File: platform/formats/csv.mdx (hunk 2 above)
Comment (Anty0) [reply platform/formats/csv.mdx:52]: I've added a CLI example, but I have a feeling we should update the supported formats in the CLI codebase.

File: blog/2024-10-08-csv-format-support-released.mdx (hunk 1 above)
Comment (Anty0) [reply blog/2024-10-08-csv-format-support-released.mdx:33]: added

**Pull request merged**